src/Wallet/Key.js

```diff
@@ -0,0 +1,16 @@
+/* global require exports BROWSER_RUNTIME */
+
+var lib;
+if (typeof BROWSER_RUNTIME != 'undefined' && BROWSER_RUNTIME) {
+    lib = require('@emurgo/cardano-serialization-lib-browser');
+} else {
+    lib = require('@emurgo/cardano-serialization-lib-nodejs');
+}
+
+exports._privateKeyFromNormalBytes = maybe => bytes => {
```

Comment:
Are we duplicating `Serialization._privateKeyFromBytes` here because of a dependency cycle?

Reply:
Nope, will fix.

But we really should not keep de-serializing functions in seiralization module. It should be in `Deserialization`.

Reply:
Made an issue for this problem https://github.com/Plutonomicon/cardano-transaction-lib/issues/560